docs/docs/api/prop.md

```diff
@@ -5,7 +5,10 @@ sidebar_label: Prop
 ---
 
 `Prop` gives access to props given to the [view](/docs/api/view)s, so that they
```

Comment:
`Prop` can also be used in producers both in relation with views or standalone, receiving some contextual information from the Engine.

Reply:
What values can a global producer (producer added directly on Engine) get as Props? Is everything in  `engineInstance.context` available as `Prop.<key>`?

Reply:
Sorry, I retract that :) I've removed actually support for passing props from the Engine instance to producers because I haven't found any use case for it. If producers need something when running at the root level they should use the state.

Would you see any value in having this?

Reply:
Can't think of any use for it based on the small apps I built using Engine.